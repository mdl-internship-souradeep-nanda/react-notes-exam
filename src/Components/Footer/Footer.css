.Footer-footer{
  flex:1;
  display: flex;
  align-items: center;
  text-align: center;
  background: #fff;
}

.Footer-center-span{
  flex:1;
}

.Footer-button{
  height: 50px;
  border: 0px;
  background: #fff;
}