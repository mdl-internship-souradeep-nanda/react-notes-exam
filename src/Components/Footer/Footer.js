import React from 'react';
import PropTypes from 'prop-types';
import './Footer.css';

function Footer(props) {
  return (
    <div className="Footer-footer">
      <strong className="Footer-center-span">
        <button className="Footer-button" onClick={props.callback}>
          {props.title}
        </button>
      </strong>
    </div>
  );
}

Footer.propTypes = {
  title: PropTypes.string.isRequired,
  callback: PropTypes.func.isRequired,
};

export default Footer;
