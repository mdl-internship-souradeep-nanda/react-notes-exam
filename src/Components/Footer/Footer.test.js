import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

test('Footer changes the class when hovered', () => {
  const callback = () => {};
  const title = 'title';
  const component = renderer.create(<Footer
    title={title}
    callback={callback}
  />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
