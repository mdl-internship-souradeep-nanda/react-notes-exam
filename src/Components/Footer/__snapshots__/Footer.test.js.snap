// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer changes the class when hovered 1`] = `
<div
  className="Footer-footer"
>
  <strong
    className="Footer-center-span"
  >
    <button
      className="Footer-button"
      onClick={[Function]}
    >
      title
    </button>
  </strong>
</div>
`;
